normandy/settings.py

```diff
@@ -12,6 +12,7 @@ class Core(Configuration):
     INSTALLED_APPS = [
         'normandy.base.apps.BaseApp',
         'normandy.classifier',
+        'normandy.control.apps.ControlApp',
```

Comment:
What's the big difference between how classifier is defined and how the rest of the apps are defined? `normandy.classifier` vs `normandy.control.apps.ControlApp`.  Should this be defined more like classifier, since it's really just going to be some views and templates built off of pre-existing apps?